### ISSUE TITLE
Using a regex file path in "rooter_patterns"?

### ISSUE THREAD

**Issue by DasOhmoff** [airblade/vim-rooter#104]
Hello, is there a way to use a regex file path with the "rooter_pattern" options?
I would use to to find a visual studio solution file in the parent directories.

I have read this issue: #85, but it only shows how to use normal filenames for the "rooter_patterns" options. Unfortunately, it does not show how to use regex patterns.
( By the way, even though the guy stopped answering in this post, your plugin and the work you did for it is very much appreciated. From all the users of your plugin: Thank you very much <3 )

In a nutshell: How can I use regex file patterns in "rooter_pattern"?
It does not necessarily have to be a regex pattern, just some way to find a solution file in the parent directories and used its directory as root. (finding '*.sln' file and using its directory as working directory)

**Comment by airblade** [comment]
So a "solution file" is any file with a `sln` extension?

**Comment by DasOhmoff** [comment]
Yes, exactly.

**Comment by airblade** [comment]
vim-rooter uses `findfile()` to look upward for files.  The help docs seem to say that `findfile()` doesn't support wildcards or regex patterns when searching up the directory tree.

Furthermore I found this [StackOverflow question](https://stackoverflow.com/q/6801679/151007) asking the same thing and coming to the same conclusion.

So unfortunately it's not possible to look for regex file patterns with vim-rooter as it currently stands.  I might change its implementation, though, to support this use case.  It would be slower but probably not enough to notice.

**Comment by DasOhmoff** [comment]
Ah ok, thats really unfortunate.
I guess there is nothing I can do about it, thanks anyway!

**Comment by airblade** [comment]
You can now (9b1fc7a) use globs – just specify `"*.sln"` in your `g:rooter_patterns`.

**Comment by DasOhmoff** [comment]
Thank you very much!
